fatoptimizer/methods.py

```diff
@@ -24,6 +24,14 @@ def check_encoding(args):
     return True
 
 
+def check_bytetype(args):
+    return all(isinstance(arg, bytes) for arg in args)
+
+
+def check_byte_or_int(*args):
+    return all(any([isinstance(arg,bytes),(isinstance(arg,int) and arg<256 and arg >=0)]) for arg in args)
```

Comment:
I just changed tox.ini to add "tox -e pep8" and run this test in Travis CI ;-) You should see the test the next time you update your PR ;-)